CoreFoundation/Preferences.subproj/CFPreferences.h

```diff
@@ -93,7 +93,7 @@ keysToFetch that are not present in the returned dictionary
 are not present in the domain.  If keysToFetch is NULL, all
 keys are fetched. */
 CF_EXPORT
-CFDictionaryRef CFPreferencesCopyMultiple(_Nullable CFArrayRef keysToFetch, CFStringRef applicationID, CFStringRef userName, CFStringRef hostName);
+_Nullable CFDictionaryRef CFPreferencesCopyMultiple(_Nullable CFArrayRef keysToFetch, CFStringRef applicationID, CFStringRef userName, CFStringRef hostName);
```

Comment:
That function does return `NULL` though and that header file has other `_Nullable` annotations, with a default of `CF_ASSUME_NONNULL_BEGIN`. So I don't see a problem with this change and it does fix an annoying warning.

Is it not a good idea to annotate as many of the function prototypes as possible over time? Or is there some downside to it?